vue-frontend/src/store/modules/authentication.js

```diff
@@ -46,14 +50,64 @@ export default {
           },
           error => {
             context.commit('update', {
-              message: error.response
+              studentMessage: error.response
                 ? error.response.data.error
                 : error.message,
               status: AuthenticationStatus.REJECTED,
             });
           },
         );
     },
+    async authenticateEducators(context, data = {}) {
+      context.commit('update', {
+        ...data,
+        status: AuthenticationStatus.PENDING,
+        educatorSignInMessage: '',
+      });
+      await context.rootState.institution.endpoints
```

Comment:
Now that I look at it, I don't think this is correct, since `await` won't do anything on an Observable. Can you just make these regular functions instead of async ones?

Reply:
You mean for all of them, `authenticate`, `authenticateEducators` and `createEducatorAccount`?

Reply:
Yeah for all of them.

Reply:
Did you mean regular instead of await ones?

Reply:
Yeah you can just remove `async` and `await` all together. Since the awaited object isn't a Promise, it resolves instantly which is meaningless in this context.